lib/improver/utilities/temporal.py

```diff
@@ -52,16 +52,17 @@
 
 
 def cycletime_to_datetime(cycletime, cycletime_format="%Y%m%dT%H%MZ"):
-    """Convert a cycletime of the format YYYYMMDDTHHMMZ into a datetime object.
+    """Convert a string representating the cycletime of the
+    format YYYYMMDDTHHMMZ into a datetime object.
 
      Args:
          cycletime (string):
              A cycletime that can be converted into a datetime using the
              cycletime_format supplied.
 
      Keyword Args:
-         cycletime_format (string):
-             String containg the appropriate directives to indicate how
+         cycletime_format (str):
+             String containing the appropriate directives to indicate how
```

Comment:
This description is incorrect to my eye. This is specifying the format of the string containing the cycletime that we wish to interpret rather than having anything to do with how we wish it to be displayed.

Reply:
I've updated this description.